hibernate-core/src/main/java/org/hibernate/engine/spi/LoadQueryInfluencers.java

```diff
@@ -182,19 +183,7 @@ public void disableFetchProfile(String name) throws UnknownProfileException {
 		enabledFetchProfileNames.remove( name );
 	}
 
-	public EntityGraph getFetchGraph() {
-		return fetchGraph;
-	}
-
-	public void setFetchGraph(final EntityGraph fetchGraph) {
-		this.fetchGraph = fetchGraph;
-	}
-
-	public EntityGraph getLoadGraph() {
-		return loadGraph;
-	}
```

Comment:
We don't expect removing these methods to cause compatibility issues?

Reply:
You could add them back if you want.  I'm not going to.  As discussed above they are misleading

Reply:
Hm, I thought I had already replied to this one.  You are possibly right.  But, in my opinion (and @aleksandarsusnjar iiuc) leaving them is confusing - they all imply that you can set both a load graph and a fetch graph.  As a compromise I'll add them back as deprecated pointing to this "confusion" fact and the correct way to do it.  Fair?

Reply:
Yep, sounds fair.

Reply:
fwiw here is the text I added locally:

````
	/**
	 * @deprecated (since 5.4) {@link #getFetchGraph}, {@link #getLoadGraph}, {@link #setFetchGraph}
	 * and {@link #setLoadGraph} (as well as JPA itself honestly) all make it very unclear that
	 * there can be only one graph applied at any one time and that graph is *either* a load or 
	 * a fetch graph.  These have all been replaced with {@link #getEffectiveEntityGraph()}.
	 * 
	 * @see EffectiveEntityGraph
	 */
````